x/auth/ante/unordered.go

```diff
@@ -1,64 +1,43 @@
 package ante
 
 import (
-	"bytes"
-	"crypto/sha256"
-	"encoding/binary"
-	"fmt"
-	"sync"
 	"time"
 
-	"github.com/cosmos/gogoproto/proto"
-
-	errorsmod "cosmossdk.io/errors"
-
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-	"github.com/cosmos/cosmos-sdk/types/tx/signing"
-	"github.com/cosmos/cosmos-sdk/x/auth/ante/unorderedtx"
 	authsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
-)
 
-// bufPool is a pool of bytes.Buffer objects to reduce memory allocations.
-var bufPool = sync.Pool{
-	New: func() interface{} {
-		return new(bytes.Buffer)
-	},
-}
+	errorsmod "cosmossdk.io/errors"
+)
 
-// DefaultSha256GasCost is the suggested default gas cost for Sha256 operations in unordered transaction handling.
-const DefaultSha256GasCost = 25
+const (
+	// MaxTimeoutDuration defines the maximum TTL a transaction can define.
+	MaxTimeoutDuration = 10 * time.Minute
```

Comment:
I think this should be configurable in the ante handler options with a default

Reply:
ok, added the ability to set options on the ante. let me know if that works